test/library/ECDSA.test.js

```diff
@@ -14,48 +14,101 @@ contract('ECDSA', function ([_, anyone]) {
     this.mock = await ECDSAMock.new();
```

Comment:
While we're at it, can you rename this to `this.ecdsa` or something more descriptive?

Reply:
done.